src/Board.py

```diff
@@ -50,6 +51,85 @@ def __init__(self, style: str = "default", size: int = 5) -> None:
         else:
             raise ValueError("Unrecognised map style")
 
+    def get(self, coords):
+        return self.data[coords[1]][coords[0]]
+
+    def get_directly_adj(self, start=(0, 0), terain_filter=None) -> list:
+        adjacent_tile_list = []
+        if terain_filter is None:
+            if start[1] > 0:
+                if start[1] % 2 == 0:  # for even rows
+                    adjacent_tile_list.append((start[0], start[1] - 1))  # top left
+                    adjacent_tile_list.append((start[0] + 1, start[1] - 1))  # top right
+                else:  # for odd rows
+                    adjacent_tile_list.append((start[0] - 1, start[1] - 1))  # top left
+                    adjacent_tile_list.append((start[0], start[1] - 1))  # top right
```

Comment:
This is not really important but it occurs to me that we could make this more concise if, at the start, we define:
```python3
row_start_offset = -(start[1] % 2)
```

And then this could just be
```suggestion
                adjacent_tile_list.append((start[0] + row_start_offset , start[1] - 1))  # top left
                adjacent_tile_list.append((start[0] + row_start_offset + 1, start[1] - 1))  # top right
```

And similarly for the bottom row